.templates/jellyfin/service.yml

```diff
@@ -0,0 +1,13 @@
+  jellyfin:
+    image: linuxserver/jellyfin
+    container_name: jellyfin
+    user: 1000:1000
+    network_mode: host
+    volumes:
+      - ./volumes/jellyfin/config:/config
+      - ./volumes/jellyfin/cache:/cache
+      - ./volumes/jellyfin/media:/media
+    devices:
+    # RPi 4 Hardware Acceleration
+    - /dev/vchiq:/dev/vchiq
```

Comment:
Does this still run even when not on a RPi/on a system that doesn't have `/dev/vchiq` without errors?

Reply:
That device is specifically for RPi 4 running Raspbian and only required for hardware acceleration. Container runs fine without that option so can be removed to accommodate other RPi versions. I only have RPi 4 so couldn't test on other version.

Reply:
I have disabled hardware acceleration by default so it can cover RPi other than 4 too.